automx2/generators/outlook.py

```diff
@@ -85,4 +90,5 @@ def client_config(self, local_part, domain_part: str, display_name: str) -> str:
             if server.type not in TYPE_MAP:
                 raise InvalidServerType(f'Invalid server type "{server.type}"')
             self.protocol_element(account, server, lookup_result.uid)
+        self._sanitise(root_element, local_part, domain_part)
```

Comment:
What if you would sanitize `account`, one line above? 🤔

Reply:
I tried it and the error is still thrown for element: 
```
      <Protocol>
        <Type>SMTP</Type>
        <Server>smtp.hostinger.com</Server>
        <Port>587</Port>
        <LoginName>%EMAILADDRESS%</LoginName>
        <SSL>off</SSL>
        <AuthRequired>on</AuthRequired>
      </Protocol>
```

Here `Protocol` has only sub-elements but doesn't have any element.text

Reply:
https://github.com/hostinger/automx2/blob/54e1eefb20ea21cddc5c62651328522b3e466302/automx2/generators/apple.py#L138-L149

`_sanitise` in apple.py generator does recursive sanitise, so should this one, too.

I guess now outlook _sanitise do not even expand and sanitise that <Protocol><LoginName>, because it does not recurse.

Reply:
```
if isinstance(element, SubElement):
    for element in value: 
        _sanitise(element, local_part, domain_part)
```

Should do the trick?

Reply:
unless element is of other type (dict, list, something else?), then replace it with appropriate type instead of SubElement